src/ForeignController.sol

```diff
@@ -105,7 +123,7 @@ contract ForeignController is AccessControl {
     /**********************************************************************************************/
 
     function transferUSDCToCCTP(uint256 usdcAmount, uint32 destinationDomain)
-        external onlyRole(RELAYER) isActive
+        external onlyRole(RELAYER) isActive rateLimited(LIMIT_USDC_TO_CCTP, usdcAmount)
     {
```

Comment:
Wondering whether it would make sense to make rate limit per domain this method interacts with. If we add more domains, one limit for all could behave weirdly.

Reply:
There are two dangers here. One is the danger of CCTP as a whole and one is danger of some compromised domain. You may be right that we want to segregate this eventually, but I think for now let's just treat CCTP as a black box to protect from global exposure to the bridge.